### PR TITLE
feat: prefer 'shot_count' to 'num_repetitions'

### PR DESCRIPTION
**Pull request by caffeine-storm**

### CODE REVIEW

File: src/anyon/qpu_interface.jl [hunk 1]
```diff
@@ -145,7 +145,7 @@ function serialize_job(circuit::QuantumCircuit,repetitions::Integer)::String
         "circuit"=>Dict{String,Any}(
                 "operations"=>Vector{Dict{String,Any}}()
             ),
-        "num_repetitions"=>repetitions
```
Comment (plerouxanyon) [inline src/anyon/qpu_interface.jl:148]: We should rename the variable `repetitions` to `shot_count` as well.

**Review by plerouxanyon — CHANGES_REQUESTED** [review]

File: src/anyon/qpu_interface.jl (hunk 1 above)
Comment (plerouxanyon) [reply src/anyon/qpu_interface.jl:148]: The `repetitions` parameter in `serialize_job` looks like it's still there.

**Review by plerouxanyon — CHANGES_REQUESTED** [review]
It seems that the `repetitions` parameter in `serialize_job` was missed, but other than that, it looks good. Overall looks good

File: src/anyon/qpu_interface.jl (hunk 1 above)
Comment (caffeine-storm) [reply src/anyon/qpu_interface.jl:148]: Oops... good catch! Fixed now.

**Review by alireza-najafi — APPROVED** [review]

**Review by alireza-najafi — APPROVED** [review]

**Review by plerouxanyon — APPROVED** [review]

**Pull request merged**